### PR TITLE
Centralize log filtering and disable log filtering if lambda fails

### PR DESCRIPTION
**Pull request by kobalski**

### CODE REVIEW

**Comment by serkan-ozal** [comment]
There is fail for `/metric.test.js`

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,26 +4,6 @@ jobs:
       docker:
         - image: circleci/node:8.11
         
-        - image: postgres:9.5-alpine
```
Comment (serkan-ozal) [inline .circleci/config.yml]: Why we are removing all the integration tests. They can fail on CI until we fix it.

File: config.json [hunk 2]
```diff
@@ -5,6 +5,6 @@
   "collectCoverageFrom": ["dist/*.js", "dist/opentracing/*.js", "dist/opentracing/instrument/*.js", "dist/opentracing/sampler/*.js", "dist/plugins/*.js", 
     "dist/plugins/data/base/*.js", "dist/plugins/data/invocation/*.js", "dist/plugins/data/metric/*.js",
     "dist/plugins/config/*.js", "dist/plugins/data/trace/*.js", "dist/plugins/error/*.js",
-    "dist/plugins/listeners/*.js", "dist/plugins/support/*.js", "index.js", "dist/plugins/integrations/*.js"
+    "dist/plugins/listeners/*.js", "dist/plugins/support/*.js", "index.js"
```
Comment (serkan-ozal) [inline config.json]: Integration tests should not be deleted as they can fail on CI until we fix it

Comment (kobalski) [reply config.json]: Disabled only ES Integration test and removed es image for now.

**Pull request merged**